src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -304,14 +304,16 @@ export const ShippingRoute: FC<ShippingProps> = props => {
             ...addressValues
           } = formValues.attributes
           requiresArtsyShipping = requiresArtsyShippingTo(addressValues.country)
+
           fulfillmentMutationValues = {
             id: props.order.internalID,
             fulfillmentType: requiresArtsyShipping
               ? "SHIP_ARTA"
               : FulfillmentType.SHIP,
             phoneNumber,
-            shipping: addressValues,
+            shipping: { ...addressValues, phoneNumber: "" },
```

Comment:
fixed test that assumes we send phone number in two places, one empty. I looked at exchange and this seems safe to ignore/fix but didn't want to bundle that into other changes on this refactor.